po/vi.po

```diff
@@ -3063,46 +3122,59 @@ msgid ""
 "of a type. The compiler does the job for us and helps us write more concise "
 "code."
 msgstr ""
+"Một điều quan trọng cần lưu ý đó là việc khai báo các biến như trên không "
+"phải là một dạng khai báo động theo kiểu \"any\" mà có thể  gán được bất kỳ "
+"dữ liệu nào. Mã máy được sinh ra từ cách khai báo này cũng giống như cách "
+"khai báo một kiểu dữ liệu cụ thể. Trình biên dịch sẽ lo liệu những việc này "
+"cho ta và giúp ta viết những đoạn mã ngắn gọn hơn."
 
 #: src/types-and-values/inference.md
 msgid ""
 "When nothing constrains the type of an integer literal, Rust defaults to "
 "`i32`. This sometimes appears as `{integer}` in error messages. Similarly, "
 "floating-point literals default to `f64`."
 msgstr ""
+"Khi không có bất cứ ràng buộc nào cho khai báo kiểu số nguyên, Rust sẽ mặc "
+"định sử dụng `i32`. Đôi khi trong thông báo lỗi biến sẽ được hiển thị dưới "
+"dạng `{integer}`. Tương tự, với các số thực dấu phẩy động, Rust mặc định là "
+"`f64`."
 
 #: src/types-and-values/inference.md
 msgid "// ERROR: no implementation for `{float} == {integer}`\n"
-msgstr ""
+msgstr "// ERROR: no implementation for `{float} == {integer}`\n"
 
 #: src/types-and-values/exercise.md
 msgid ""
 "The first and second Fibonacci numbers are both `1`. For n>2, the n'th "
 "Fibonacci number is calculated recursively as the sum of the n-1'th and "
 "n-2'th Fibonacci numbers."
 msgstr ""
+"Hai số Fibonacci đầu tiên mang giá trị `1`. Với n > 2, số Fibonacci thứ n sẽ "
```

Comment:
Btw, this statement seems like it was not rendered when I was running the mdbook. Instead, it showed the following statements: 

```
The Fibonacci sequence begins with [0,1]. For n>1, the n’th Fibonacci number is calculated recursively as the sum of the n-1’th and n-2’th Fibonacci numbers.
```

![image](https://github.com/google/comprehensive-rust/assets/68913255/4d653c96-5511-4967-acf3-ca952230f6f9)

@daivinhtran Could you fix it?

Reply:
@khoaxuantu can you re-run and tell me if it still occurs?

Reply:
> It looks good rn. Do you still have problems running mdbook for the first two lines of the docs?

@qu-ngx Yes, it still occurs. The screenshot that I took was when I tried to run mdbook locally, btw.

Reply:
@khoaxuantu hmm it seems like something must have occurred.